### PR TITLE
Fix ordering of import dependencies once and for all

### LINKED ISSUE #284: vulcanize alters script execution order unexpectedly
vulcanize (1.12.3) unexpectedly alters the order of script execution if an imported document has a script in the `<body>`. Example: http://pastebin.com/raw.php?i=1BPLc5iA

The unvulcanized `in.html`will run the first script first, but loading the output of `vulcanize in.html` will run the second script first.

Is this expected behavior, or a bug?

### PR DESCRIPTION
**Pull request by azakus**
All import portions, `<head>` and `<body>` will go into the body of the main
document. This is the only way to ensure that intra-import dependencies and
inter-import dependencies line up correctly.

Fixes #284

### CODE REVIEW

**Comment by azakus** [comment]
PTAL @garlicnation @rictic

File: lib/vulcan.js [hunk 1]
```diff
@@ -391,6 +381,27 @@ Vulcan.prototype = {
         return this.inlineCss(docObj.doc, docObj.href);
       }.bind(this));
     }
+
+    if (this.stripComments) {
```
Comment (rictic) [inline lib/vulcan.js]: Worth adding a test for stripComments

Comment (azakus) [inline lib/vulcan.js]: There is a check at https://github.com/Polymer/vulcanize/pull/293/files#diff-c1129c8b045390789fa8ff62f2c6b4a9R598, but I made it more explicit in 28a7812

**Comment by rictic** [comment]
LGTM!

**Comment by azakus** [comment]
@sorvell @kevinpschaaf PTAL

**Pull request merged**